articles/active-directory-b2c/active-directory-b2c-ui-customization-custom.md

```diff
@@ -74,18 +74,19 @@ To host this HTML content in Blob storage, do the following:
 
 To create a public container in Blob storage, do the following:
 
-1. Click the **Overview** tab.
-2. Click **Container**.
-3. For **Name**, type **$root**.
+1. Click the **Blobs** tab under the Blob service menu item group in menu.
+2. Click **+Container**.
+3. For **Name**, type **root**. (You can give it a name of your choice like *wingtiptoys*.)
```

Comment:
```suggestion
3. For **Name**, enter *root*. This can be a name of your choosing, for example *wingtiptoys*, but we use *root* in this example for simplicity.
```